docker-testing/in-container-test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# This is the test script that runs in the containers themselves.
+
+WEST=/west
+
+die() {
+    if [ ! -z "$@" ]; then
```

Comment:
"$@" expands to multiple strings. That's not what you want here
http://mywiki.wooledge.org/BashPitfalls#for_arg_in_.24.2A
http://mywiki.wooledge.org/BashGuide/Parameters#Special_Parameters_and_Variables


```suggestion
    if [ -n "$*" ]; then
```

Or even just:
```suggestion
    if [ -n "$1" ]; then
```

Or `if [ $# -eq 0 ]`

Then for consistency:

`echo "error: $*" >&2`

Reply:
BTW you don't HAVE to `printf 'error: %s' "$var"`, you can still do `printf "error: $var"` if you want. The well-tested `die()` function I recommended earlier supports either indifferently.